### DISCUSSION TITLE
Overlapping panels

### DISCUSSION THREAD

**Discussion by SamSandq** [frang75/nappgui_src#77] [Ideas]
I’m writing a photo management app, and would like to see photos (imageviews) with overlaid data (stars, name, tags).

I haven’t found a way to achieve this in NAppGui on the mac, but know it’s possible using views in Swift with the same position, and semitransparent background (maybe 30% opacity) in labels.

Any ideas?

**Comment by frang75** [comment]
Hi @SamSandq. You can doit with custom views, where you can draw everything (images, text, shapes, etc).
Please review the code of this demo (left side): https://nappgui.com/en/howto/drawimg.html
Also, if you want to separate the main drawing for the overlay, you can use the new 1.4 feature. https://nappgui.com/en/gui/view.html#h3

**Comment by SamSandq** [comment]
Hi,

I will try that - wasn't aware that you could put images on views too. I
will also upgrade to 1.4, since it's available.

Any news on the upcoming Tree (=outline) control?

Cheers,

---------------------------------
Sam

PS: ¡Y muchas gracias por responderme tan rápido!

Den tors 28 dec. 2023 kl 11:03 skrev Francisco García Collado <
***@***.***>:

**Reply by frang75** [reply]
The TreeView features for the TableView control is planned for NAppGUI 1.4.2 (March/April of incoming 2024).
